Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SpannerTestDatabase.cs

```diff
@@ -118,8 +120,15 @@ private async Task MaybeCreateInstanceAsync()
             // Check if the instance exists and if not create it.
             var adminClientBuilder = new InstanceAdminClientBuilder
             {
-                EmulatorDetection = EmulatorDetection.EmulatorOrProduction
+                EmulatorDetection = EmulatorDetection.EmulatorOrProduction,
+                GrpcAdapter = GrpcNetClientAdapter.Default,
             };
+            if (Environment.GetEnvironmentVariable("SPANNER_EMULATOR_HOST") != null)
```

Comment:
Note that in the next major version of GAX and the Spanner libraries (coming soon), Grpc.Net.Client will be the default gRPC implementation - this code will need to change.

(I'm assuming the new Spanner version will come out before the EFCore 6 support... I really think it's worth waiting for that.)

Reply:
What is 'soon'?. I would prefer that this PR does not drag on for too long, as the time needed to get deeply invested in this again in let's say a month is relatively high. Changing this (test) code a little later should be relatively easy.

Or will the new GAX and Spanner libraries require changes to the 'normal' usage of the libraries as well?

Reply:
FYI: Google.Cloud.Spanner.Data (etc) are now being released as GA, version 4.0.0. (By the time you read this, the release process is likely to have completed.) So now would be a good time to update.